packages/aws-cdk/lib/cdk-toolkit.ts

```diff
@@ -304,10 +310,12 @@ export class CdkToolkit {
     // --------------                --------  'cdk deploy' done  --------------  'cdk deploy' done  --------------
     let latch: 'pre-ready' | 'open' | 'deploying' | 'queued' = 'pre-ready';
 
+    const cloudWatchLogMonitor = options.traceLogs ? new CloudWatchLogEventMonitor() : undefined;
```

Comment:
cc @skinny85

Reply:
We don't have currently any continuous running command in our CLI beyond `watch`, so I don't see where it would fit besides `watch`, and it slots into `watch` beautifully.

If you want to create a separate command for this (`cdk logs`, or something), I think that's reasonable, but that's a separate task and PR.